test/config/auth/policy.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: authentication.istio.io/v1alpha1
```

Comment:
`kubectl apply -f ` is not recursive without option, so we need to explicitly apply this policy in e2e-common.sh  like https://github.com/knative/serving/blob/28f2a2460ee8fe89cca025d0831380ea7fe0df04/test/e2e-common.sh#L436

But could you please consider this https://github.com/knative/serving/pull/6505/files#r365568314 first?

Reply:
ok.

Reply:
if i add the policy to serving-tests or serving-tests-alt, it would affect the other tests run on these ns as it would require token to call the service end point.

Reply:
I see. I wondered we can set `excludedPaths.exact: /` and use another path for the test. But as per your report https://github.com/knative/serving/issues/5918, we want to make sure that `/` is not necessary to be exluded. Then, it seems that we need to create new namespace for this.